recipe/patches/0019-do-not-use-cxx-icu-api-in-tests.patch

```diff
@@ -0,0 +1,21 @@
+From 13ef056a98f0e49992367609cee952a830086fa9 Mon Sep 17 00:00:00 2001
+From: Silvio Traversaro <silvio@traversaro.it>
+Date: Fri, 13 Sep 2024 18:53:10 +0200
+Subject: [PATCH] Define U_SHOW_CPLUSPLUS_API to 0 in ICU test
+
+---
+ src/corelib/configure.json | 1 +
+ 1 file changed, 1 insertion(+)
+
+diff --git a/src/corelib/configure.json b/src/corelib/configure.json
+index a9025a2dd29..491f0dcd90a 100644
+--- a/src/corelib/configure.json
```

Comment:
Is this supposed to be?

'qtbase/src/corelib/configure.json'

Reply:
Thanks, done in https://github.com/conda-forge/qt-main-feedstock/pull/288/commits/75ae665ce2cdf6b2f63c89eda23ccdc0c0c2cc01 .